lib/auth/pages/sign_in/form.dart

```diff
@@ -110,7 +110,14 @@ class _SignInFormState extends State<SignInForm> {
           isLoading: _isSigningInAsDemoStudent,
           onPressed: () async {
             setState(() => _isSigningInAsDemoStudent = true);
-            await demoSignIn('demo-lehrer@schul-cloud.org', 'schulcloud');
+            try {
+              await demoSignIn('demo-lehrer@schul-cloud.org', 'schulcloud');
+            } catch (e) {
+              logger.e(exceptionMessage(e, context));
+              context.scaffold.showSnackBar(SnackBar(
+                content: Text(exceptionMessage(e, context)),
+              ));
+            }
```

Comment:
This change should also be applied to the other demo sign-in. Also, I just noticed that the "Sign in as demo student" button signs you in as a demo teacher and vice versa — that might not be the best UX 😂